plugins/http_plugin/http_plugin.cpp

```diff
@@ -392,6 +392,10 @@ namespace eosio {
          cfg.add_options()
             ("unix-socket-path", bpo::value<string>()->default_value(current_http_plugin_defaults.default_unix_socket_path),
              "The filename (relative to data-dir) to create a unix socket for HTTP RPC; set blank to disable.");
+      else
```

Comment:
unfortunately I had to do the else here because `default_value()` balked at an empty string 🤷‍♂